docs/release-info/release-notes.md

```diff
@@ -57,40 +57,284 @@ For tips about how to write a good release note, see [Release notes](https://git
+- UNNEST is now supported for autocomplete suggestions [#17521](https://github.com/apache/druid/pull/17521)
+- Tables now support less than and greater than filters [#17521](https://github.com/apache/druid/pull/17521)
+- You can now resize the side panels in the Query view [#17387](https://github.com/apache/druid/pull/17387)
+- Added the `expectedLoadTimeMillis` segment loading metric to the web console [#17359](https://github.com/apache/druid/pull/17359)
+
 ### Ingestion
 
+#### Numbers for CSV and TSV input formats
+
+Use the new optional config `tryParseNumbers` for CSV and TSV input formats to control how numbers are treated. If enabled, any numbers present in the input will be parsed in the following manner:
+
+- long data type for integer types and 
+- double for floating-point numbers
+ 
+By default, this configuration is set to false, so numeric strings will be treated as strings.
+
+[#17082](https://github.com/apache/druid/pull/17082)
+
+#### Other ingestion improvements
+
+- Reduce the direct memory requirement on non-query processing tasks by not reserving query buffers for them [#16887](https://github.com/apache/druid/pull/16887)
+- JSON-based and SQL-based ingestion now support request headers when using an HTTP input source [#16974](https://github.com/apache/druid/pull/16974)
+ 
 #### SQL-based ingestion
 
 ##### Other SQL-based ingestion improvements
 
+- SQL-based ingestion now supports dynamic parameters for queries besides SELECT queries, such as REPLACE [#17126](https://github.com/apache/druid/pull/17126)
+- Improved thread names to include the stage ID and worker number to help with troubleshooting [#17324](https://github.com/apache/druid/pull/17324)
+
 #### Streaming ingestion
 
+##### Control how many segments get merged for publishing
+
+You can now use the `maxColumsnToMerge` property in your supervisor spec to specify the number of segments to merge in a single phase when merging segments for publishing. This limit affects the total number of columns present in a set of segments to merge. If the limit is exceeded, segment merging occurs in multiple phases. Druid merges at least 2 segments each phase, regardless of this setting.
+
+[#17030](https://github.com/apache/druid/pull/17030)
+
 ##### Other streaming ingestion improvements
 
+- Druid now properly supports early/late rejection periods when `stopTasksCount` is configured and streaming tasks run longer than the configured task duration [#17442](https://github.com/apache/druid/pull/17442)
+- Improved segment publishing when resubmitting supervisors or when task publishing takes a long time [#17509](https://github.com/apache/druid/pull/17509)
+
 ### Querying
 
+#### Window queries
+
+The following fields are deprecated for window queries that use the MSQ task engine: `maxRowsMaterializedInWindow` and `partitionColumnNames`. They will be removed in a future release.
+
+[#17433](https://github.com/apache/druid/pull/17433)
+
+
+
```

Comment:
Missing content?

Reply:
This was the join hints that got moved to the top